wurst/network/Network.wurst

```diff
@@ -0,0 +1,922 @@
+package Network
+
+import SyncSimple
+import Table
+import ErrorHandling
+import MoriUtil
+
+// max amount to be written/read/synced per an execution to avoid hitting the OP limit
+// this value can be higher if Wurst is compiled without stack traces and with all optimizations turned on
+// and even higher with safety checks off
```

Comment:
The above two comment are of the form "optimiations, safety checks, and stack traces affect suitable op-limit values". I don't think they add much value here.

Reply:
I think these are important considerations for those who may feel like they are experiencing performance issues, and are not necessarily obvious to everyone. Especially not obvious what this constant is for, and why you may want to adjust it, hence, the comment.

Reply:
May I suggest

```
// Max amount to be written/read/synced per an execution, to avoid hitting the op-limit.
// Beware of changes in maximum value as compiler settings are adjusted.
```